src/libraries/MathLib.sol

```diff
@@ -8,33 +8,33 @@ uint256 constant WAD = 1e18;
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic.
 library MathLib {
-    /// @dev (x * y) / WAD rounded down.
+    /// @dev Returns (`x` * `y`) / `WAD` rounded down.
     function wMulDown(uint256 x, uint256 y) internal pure returns (uint256) {
         return mulDivDown(x, y, WAD);
     }
 
-    /// @dev (x * WAD) / y rounded down.
+    /// @dev Returns (`x` * `WAD`) / `y` rounded down.
```

Comment:
```suggestion
    /// @dev Returns (`x` * WAD) / `y` rounded down.
```
not sure if we should do this actually. So the idea of backticks is to replace the expression dynamically with a concrete value, as explained [here](https://docs.soliditylang.org/en/v0.8.11/natspec-format.html#dynamic-expressions). But after seeing how it is used on this particular line, it feels weird to not do it for every "verbatim" (code-like) expressions

Reply:
The decision taken is to allow to have backticks on global variables, since it's not clear how expression are replaced dynamically anyway